### PR TITLE
Proposed some additional paragraphs to hammer home a few points

### PR DESCRIPTION
**Pull request by gendal**
I've proposed a few extra paragraphs to hammer home why 'teach corda how to check the inclusion of an event but rely on the validator signatures for finality' is actually pretty cool

### CODE REVIEW

File: docs/adhara/cross_chain_proofs.md [hunk 1]
```diff
@@ -11,15 +11,19 @@ If we go too far to the right, we risk interlinking two systems and the interope
 
 So the real question is, how do we get to a solution that can scale, but doesn't have a single point of failure?
 
-## Trust and authority in distributed system interoperability##
+In what follows, it is helpful to think of the problem from the perspective of a party _who is not a member of the network in question_ who wants to know if an event they occur about has happened. That is: given a data structure purporting to be a block from the remote chain, 1) does it perform the action I care about and 2) if so, has the block actually been confirmed as part of the settled history of the chain?
```
Comment (coeniebeyers) [inline docs/adhara/cross_chain_proofs.md:14]:
```suggestion
In what follows, it is helpful to think of the problem from the perspective of a party _who is not a member of the network in question_ who wants to know if an event they care about has happened. That is: given a data structure purporting to be a block from the remote chain, 1) does it perform the action I care about and 2) if so, has the block actually been confirmed as part of the settled history of the chain?
```

File: docs/adhara/cross_chain_proofs.md [hunk 2]
```diff
@@ -11,15 +11,19 @@ If we go too far to the right, we risk interlinking two systems and the interope
 
 So the real question is, how do we get to a solution that can scale, but doesn't have a single point of failure?
 
-## Trust and authority in distributed system interoperability##
```
Comment (MicheleCullinan) [inline docs/adhara/cross_chain_proofs.md:14]: ## Trust and authority in distributed system interoperability ##

File: docs/adhara/cross_chain_proofs.md [hunk 3]
```diff
@@ -11,15 +11,19 @@ If we go too far to the right, we risk interlinking two systems and the interope
...
 
-In a permissioned network using a proof of authority consensus mechanism like QBFT, a chosen set of authorities/validators get to propose the next block of transactions, however it is still up to each node to verify and execute the block of transactions on their local copy of the blockchain. In such a system more trust is given to the authorities/validators, however each node can still verify each block and transaction, and there is a drastic reduction in latency and immediate finality of blocks can be achieved. 
+In a permissioned network using a proof of authority consensus mechanism like QBFT, a chosen set of authorities/validators get to propose the next block of transactions, however it is still up to each node to verify and execute the block of transactions on their local copy of the blockchain. In such a system more trust is given to the authorities/validators, however each node can still verify each block and transaction, and there is a drastic reduction in latency and immediate finality of blocks can be achieved. Importantly, the validators are also expected to ensure the correctness of each block.
```
Comment (petermunnings) [inline docs/adhara/cross_chain_proofs.md:20]: Possible:  "Importantly, the validators are expected to sign the block once they have verified the block"

**Review by coeniebeyers — APPROVED** [review]

**Review by petermunnings — APPROVED** [review]

File: docs/adhara/cross_chain_proofs.md [hunk 4]
```diff
@@ -15,7 +15,7 @@ In what follows, it is helpful to think of the problem from the perspective of a
...
+In a permissioned network using a proof of authority consensus mechanism like QBFT, a chosen set of authorities/validators get to propose the next block of transactions, however it is still up to each node to verify and execute the block of transactions on their local copy of the blockchain. In such a system more trust is given to the authorities/validators, however each node can still verify each block and transaction, and there is a drastic reduction in latency and immediate finality of blocks can be achieved. Importantly, the validators are expected to sign each block they have verified
```
Comment (petermunnings) [inline docs/adhara/cross_chain_proofs.md:18]: excellent

**Review by MarkOldfield — APPROVED** [review]

**Pull request merged**